package.json

```diff
@@ -104,7 +104,7 @@
     "shpjs": "^3.5.0",
     "simple-oauth2": "git+https://github.com/bilts/simple-oauth2.git#fix-encoding",
     "source-map-support": "^0.5.19",
-    "sqlite3": "^4.2.0",
+    "sqlite3": "^5.0.2",
```

Comment:
What required these changes, removing pre-gyp and updating sqlite3?

Reply:
`node-pre-gyp` is no longer supported, I switched it over to `@mapbox/node-pre-gyp`. No building from source was needed with the combination of changes for node-pre-gyp, npm, and this version of sqlite.